### PR TITLE
TM-3608: Update/panel agenda UI

### PR DESCRIPTION
**Pull request by pveland621**
ACs:
1. Verify all fields are coming through correctly from dummy data to AgendaItemRow
2. Verify remarks pills are displaying correctly

@ scott and elizabeth, cory brought up a good point on whether or not i should set up multiple rows to help show the color changing tickets in the future, but to do that it would be something around 200 lines of dummy data in the code for 3 rows - requesting a final call from you guys

There are several differences from my screenshot and the ticket's - all missing work from the ticket screenshot is captured in other stories.

Ticket Screenshot:
image

Final Implementation:
image

### CODE REVIEW

**Comment by scott062** [comment]
Nice job, lets move the dummy data to an action file and setup the redux loop to get this data. We'll just want to make sure we're calling the "action" etc. to populate this page

File: src/Components/Panel/PanelMeetingAgendas/PanelMeetingAgendas.jsx [hunk 1]
```diff
@@ -33,6 +33,7 @@ const PanelMeetingAgendas = ({ isCDO }) => {
 
   const userSelections = useSelector(state => state.panelMeetingAgendasSelections);
   const genericFilters = useSelector(state => state.filters);
+  const agenda = useSelector(state => state.panelMeetingAgendas); // it's panelMeetingAgenda
```
Comment (SidelineCory24) [inline src/Components/Panel/PanelMeetingAgendas/PanelMeetingAgendas.jsx:36]: can remove this comment

File: src/Components/Panel/PanelMeetingAgendas/PanelMeetingAgendas.jsx [hunk 2]
```diff
@@ -412,6 +413,24 @@ const PanelMeetingAgendas = ({ isCDO }) => {
...
+                        key={result.id}
+                        isCDO={isCDO}
+                        isAIHView
+                        agenda={result}
+                      />
+                    ))
+                  }
+                </div>
+              }
+            </>
```
Comment (elizabeth-jimenez) [inline src/Components/Panel/PanelMeetingAgendas/PanelMeetingAgendas.jsx:432]: Might not need the extra  `<></>`

File: src/actions/panelMeetingAgendas.js [hunk 3]
```diff
@@ -59,42 +113,13 @@ export function panelMeetingAgendasExport(query = {}) {
     });
 }
 
-export function panelMeetingAgendasFetchData(query = {}) {
+export function panelMeetingAgendasFetchData() {
   return (dispatch) => {
-    if (cancelPanelMeetingAgendas) { cancelPanelMeetingAgendas('cancel'); }
     batch(() => {
-      dispatch(panelMeetingAgendasFetchDataLoading(true));
+      dispatch(panelMeetingAgendasFetchDataSuccess(dummyAgenda));
```
Comment (elizabeth-jimenez) [inline src/actions/panelMeetingAgendas.js:119]: ❤️

File: src/Components/Panel/PanelMeetingAgendas/PanelMeetingAgendas.jsx (hunk 1 above)
Comment (pveland621) [reply src/Components/Panel/PanelMeetingAgendas/PanelMeetingAgendas.jsx:36]: note for myself 😅  i'll clean

File: src/Components/Panel/PanelMeetingAgendas/PanelMeetingAgendas.jsx (hunk 2 above)
Comment (pveland621) [reply src/Components/Panel/PanelMeetingAgendas/PanelMeetingAgendas.jsx]: done!

**Review by scott062 — APPROVED** [review]
👍

**Pull request merged**